releases/release-1.26/release-notes/maps/pr-112741-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 112741
+releasenote:
+  text: Expose health check SLI metrics on `metrics/slis` for apiserver.
```

Comment:
`Exposed health check SLI metrics on `metrics/slis` for apiserver.`

Reply:
/Ack